umm-spec-lib/test/cmr/umm_spec/test/xml_to_umm_mappings/iso19115_2/spatial.clj

```diff
@@ -39,3 +38,18 @@
                   (xpath/context)
                   (spatial/parse-coordinate-system))]
     (is (= "GEODETIC" value))))
+
+(deftest test-coordinate-system-parsing-large-num
+  "Testing the ability to handle parsing a number that exceeds the max integer value"
+
+  (let [value (spatial/parse-horizontal-data-resolutions
+               (slurp (io/resource "example-data/iso19115/artificial_test_data_large_num.xml")))]
+    (is (= 2147483648 (-> value :GriddedRangeResolutions first :MaximumYDimension)))))
+
+(deftest test-coordinate-system-parsing-invalid
+  "Testing the caught error when parsing an invalid number"
+
+  (let [value (try (spatial/parse-horizontal-data-resolutions
```

Comment:
This test is inadequate -- it will pass with even the previous syntax error code. Use `cmr.common.test.test-util/assert-exception-thrown-with-errors` to test exceptions thrown.